app/pulp/app/settings.py

```diff
@@ -28,6 +29,12 @@
 # SECURITY WARNING: don't run with debug turned on in production!
 DEBUG = True
 
+# Silenced warnings
+# - fields.W342 is the warning for having "unique=True" on a ForeignKeyField, which is something
+#   we explicitly want, so we can enforce a single Importer per-repository in 3.0, but easily
+#   allow for multiple importers per-repository in a later version by lifting the contraint.
+SILENCED_SYSTEM_CHECKS = ["fields.W342"]
```

Comment:
My thinking is that I should only create the story if this gets approved, so I think we agree, and I'll get that written up once we merge.